### ISSUE TITLE
Error when try to run build 

### ISSUE THREAD

**Issue by jrussumbrella** [bradgarropy/nextjs-firebase#1]
![Screenshot from 2019-08-06 18-04-43](https://user-images.githubusercontent.com/39219294/62531618-5b107380-b875-11e9-95fd-98648c2a9015.png)

**Comment by bradgarropy** [comment]
I can try to reproduce this later tonight.
In the meantime, have you tried running `npm rebuild` like the error message says?

**Comment by bradgarropy** [comment]
I got a chance to reproduce just now, and I'm seeing this with `next@8.1.0` as well.

**Comment by jrussumbrella** [comment]
Yes sir everything works well with version of 8 but when I try to update with the latest version of next which is 9, then try to npm run build I now got an error. I also tried running npm rebuild but still same error exist.

**Comment by jrussumbrella** [comment]
Have you tried updating the version of next in your package.json and run npm run build?
